### PR TITLE
Variant Control pre-work

### PR DESCRIPTION
**Pull request by anthonylasso**
# Description

This PR serves as a bit of cleanup before implementing the main variant rework. 

It cleans up parts of the tags, entity view, and code view. 

`Jest Tests`
image

### Select type(s) of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

# Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have fixed any merge conflicts

### CODE REVIEW

File: dashboard/src/components/entitypage/EntityPageView.js [hunk 1]
```diff
@@ -39,7 +40,7 @@ SyntaxHighlighter.registerLanguage('json', json);
 export function getFormattedSQL(sqlString = '') {
   let stringResult = sqlString;
   try {
-    stringResult = format(sqlString.replace('{{', '').replace('}}', ''), {
+    stringResult = format(sqlString.replace('{{ ', '"').replace(' }}', '"'), {
```
Comment (sdreyer) [inline dashboard/src/components/entitypage/EntityPageView.js:43]: Do we want this? Will it get confusing if a table actually has quotes?

**Review by sdreyer — APPROVED** [review]

File: dashboard/src/components/entitypage/statusChip.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Chip } from '@mui/material';
+import React from 'react';
+
+export default function StatusChip({ status = '' }) {
+  console.log(status);
+  const chipMap = {
+    NO_STATUS: '#FFFFFF',
+    CREATED: '#FFFFFF',
```
Comment (ahmadnazeri) [inline dashboard/src/components/entitypage/statusChip.js:8]: just curious why `NO_STATUS` and `CREATED` are both black?

**Review by ahmadnazeri — APPROVED** [review]

File: dashboard/src/components/entitypage/EntityPageView.js (hunk 1 above)
Comment (anthonylasso) [reply dashboard/src/components/entitypage/EntityPageView.js:43]: that is a possibility, but without these quotes, a table like `transactions.2024-03-27t14-53-01` will cause the sql formatter to fail and it spits out an ugly one line string.

File: dashboard/src/components/entitypage/statusChip.js (hunk 2 above)
Comment (anthonylasso) [reply dashboard/src/components/entitypage/statusChip.js:8]: good catch (both are white) I'll quickly just set something for `created`

File: dashboard/src/components/entitypage/EntityPageView.js (hunk 1 above)
Comment (anthonylasso) [reply dashboard/src/components/entitypage/EntityPageView.js:43]: worst case, if a table does have quotes (like in Oracle or something) we can just check for pairs and remove one

**Pull request merged**